inc/class-ad-server.php

```diff
@@ -118,7 +122,7 @@ public function setup() {
 			$this->ad_servers = apply_filters(
 				'ad_layers_ad_servers',
 				[
-					'Ad_Layers_DFP' => '\Ad_Layers\Ad_Servers\Ad_Layers_DFP',
+					'Ad_Layers_DFP' => dirname( __FILE__ ) . '/ad-servers/class-ad-layers-dfp.php',
 				]
 			);
 
```

Comment:
This whole thing could (should?) be replaced with a PHP autoloader instead. Feel free to ignore.

Reply:
Thanks for the advice and the backchannel knowledge dump. Goint to note this as a second step for now, but reference this in a new GitHub issue #83